pangeo-notebook/environment.yml

```
@@ -6,6 +6,7 @@ dependencies:
  - pangeo-notebook=2022.01.31
  - pip=20
  - adlfs
+ - argopy >= 0.1.9
```

```suggestion
 - argopy
```

Once the new argopy release is up on conda forge, let's go back to this.